src/frontend-web/src/App/Routes/anon.routes.tsx

```diff
@@ -1,13 +1,15 @@
 import { Navigate, Route, Routes } from "react-router-dom";
 import Signin from "../Pages/SignIn";
 import SideBar from "../Components/SideBar";
+import Signup from "../Pages/SignUp";
 
 const AnonRoutes = () => {
   return (
     <SideBar>
       <Routes>
         <Route path="/" element={<Navigate replace to="/login" />} />
         <Route path="/login" element={<Signin />} />
+        <Route path="/registro" element={<Signup />} />
```

Comment:
Faz sentido colocar a tela de registro na rota autenticada?

Reply:
Não, por isso coloquei em anon. Não é aí que são definidas rotas não autenticadas? Até pq não faria sentindo eu estar autenticado pra me registrar ou logar

Reply:
comentei no arquivo errado mas se você for ver, ele está nas appRoutes tbm.

Reply:
Num tá